package.json

```diff
@@ -76,7 +81,7 @@
   "scripts": {
     "test": "jest",
     "start": "ts-node src/index.ts",
-    "start:watch": "nodemon -x npm start",
+    "start:watch": "npx nodemon -x npm start",
```

Comment:
What is the benefit of using npx?

The reason I use it directly is I like to have it locked down by the package json, when you use npx you don't know what version you're going to get